iceoryx_hoofs/include/iceoryx_hoofs/cxx/helplets.hpp

```diff
@@ -64,17 +64,17 @@ struct BestFittingTypeImpl<true, true, false>
 {
     using Type_t = uint32_t;
 };
-
-constexpr char ASCII_A = 'a';
-constexpr char ASCII_Z = 'z';
-constexpr char ASCII_CAPITAL_A = 'A';
-constexpr char ASCII_CAPITAL_Z = 'Z';
-constexpr char ASCII_0 = '0';
-constexpr char ASCII_9 = '9';
-constexpr char ASCII_MINUS = '-';
-constexpr char ASCII_DOT = '.';
-constexpr char ASCII_COLON = ':';
-constexpr char ASCII_UNDERSCORE = '_';
+// AXIVION DISABLE STYLE AutosarC++19_03-A3.9.1: Basic numerical type of char shall be used
```

Comment:
Can we close this style section earlier? We should avoid long sections as we might suppress actual issues (bugs).